schema/proto3/port.proto

```diff
@@ -53,16 +58,14 @@ message PortConfiguration {
         string mac_address = 2;
     }
 
-    message ExtraDhcpOption {
-        string name = 1;
-        string value = 2;
-    }
-
-    HostInfo host_info = 9;
+    HostInfo host_info = 9; // FIXME: will be used to populate neighbor info only
     repeated FixedIp fixed_ips = 10;
     repeated SecurityGroupId security_group_ids = 11;
     repeated AllowAddressPair allow_address_pairs = 12;
-    repeated ExtraDhcpOption extra_dhcp_options = 13;
```

Comment:
Is dhcp options completely covered by the new dhcp.proto

Reply:
yes, that would include DNS server(s) list also.